lib/grr.js

```diff
@@ -1220,6 +1273,28 @@ function grrDelete(opts, cb) {
     ]}, cb);
 }
 
+/*
+ * List open CRs for the current repository. Intended for use by engineers
+ * who want to help cleaning up any CR backlog that may exist.
+ */
+function grrList(opts, cb) {
+    assert.object(opts, 'opts');
+    assert.object(opts.log, 'opts.log');
+    assert.func(cb, 'cb');
+
+    var context = {
+        log: opts.log,
+    };
+
+    vasync.pipeline({arg: context, funcs: [
+        stepGetGrrConfig,
```

Comment:
add a `stepGrrCache` line here per https://github.com/joyent/grr/pull/6#discussion_r209687596 to avoid

```
$ timgrr -L
/Users/trentm/src/timfoster-grr/lib/grr.js:176
    } else if (arg.grrCache.gerritUsername) {
                           ^

TypeError: Cannot read property 'gerritUsername' of undefined
    at Object.stepGetGerritUsername [as func] (/Users/trentm/src/timfoster-grr/lib/grr.js:176:28)
    at Immediate._onImmediate (/Users/trentm/src/timfoster-grr/node_modules/vasync/lib/vasync.js:213:20)
    at processImmediate [as _immediateCallback] (timers.js:396:17)
```

at least for my case.

Reply:
Will do - I didn't hit this because I had this in ~/.grrrc:
```
[gerrit]
username = "timfoster"
```
commenting that out, and I hit the bug as expected - thanks.